packages/utils/sets-utils/src/cell-set-utils.js

```diff
@@ -559,6 +559,76 @@ export function filterNode(node, prevPath, filterPath) {
   };
 }
 
+/**
+ * Two arrays of paths are equal (contain the same elements).
+ * @param {Array<string[]>} a Set of paths.
+ * @param {Array<string[]>} b Other set of paths.
+ * @returns {boolean} Whether the two sets are equivalent.
+ */
+export function isEqualSet(a, b) {
+  if (a.length === b.length) {
+    const aSet = new Set(a);
+    aSet.forEach((aVal) => {
+      b.forEach((bVal) => {
+        if (isEqual(aVal, bVal)) {
+          aSet.delete(aVal);
+        }
+      });
+    });
+    if (aSet.size === 0) {
+      return true;
+    }
+  }
+  return false;
+}
+
+/**
+ * One set contains part of another.
+ * @param {Array<string[]>} a Set of paths.
+ * @param {Array<string[]>} b Other set of paths.
+ * @returns {boolean} Whether set B is partially contained in set A.
+ */
+export function isPartialSet(a, b) {
+  // b is partially contained in a.
+  if (a.length > 0 && b.length > 0) {
```

Comment:
Like, `Set` is not used here but is below so not sure.

Reply:
`Set` is just used for the internal implementation

Reply:
Taking advantage of the fact that the arrays of strings will be represented in the `Set` by the reference to their array object. Could be implemented many other ways.